### ISSUE TITLE
Opensource the application code

### ISSUE THREAD

**Issue by eusebium** [HappenApps/Quiver#1485]
Hi,
if you are still alive, can you release to the community the Quiver code?
Is a great application with lack of updates and support.

Thank You

**Comment by mm4nn** [comment]
Yes. Please either provide proof of life (of the application's development) We can see that there is activity on your GitHub profile for private repositories but that does not mean you are working on Quiver--you can be working on other things.

So please at least let us know that there is a chance that you are working on Quiver updates **OR** _please please please_ open source the code so that the community may take over. I have been searching for an equivalent and nothing has come close... Evernote, Notion, MWeb, etc... nope, nope, nope, nope.

**Comment by sb-** [comment]
https://github.com/HappenApps/Quiver/issues/1516

We are reviving it
